core/src/subgraph/instance_manager.rs

```diff
@@ -929,6 +929,22 @@ where
         info!(&logger, "Applying {} entity operation(s)", mods.len());
     }
 
+    if !block_state.deterministic_errors.is_empty() {
+        info!(
+            &logger,
+            "Applying {} error(s)",
+            block_state.deterministic_errors.len()
+        );
+    }
+
+    for e in &block_state.deterministic_errors {
+        error!(&logger, "Subgraph error";
+            "error" => e.to_string(),
+            "id" => ctx.inputs.deployment_id.to_string(),
+            "code" => LogCode::SubgraphSyncingFailure
```

Comment:
Nice; I didn't know we had `LogCode` - we should probably use it more pervasively

Reply:
Yhea Jorge introduced it